src/cpu_regs.h

```diff
@@ -123,3 +123,66 @@
 
 #define SYS_IMP_APL_UPMSR sys_reg(3, 7, 15, 6, 4)
 #define UPMSR_IACT        (BIT(0))
+
+/* SPRR and GXF registers */
+#define SYS_SPRR_CONFIG_EL1       sys_reg(3, 6, 15, 1, 0)
```

Comment:
Please follow the SYS_IMP_APL_ convention here (I know it's verbose, but it's what we're using in Linux, so let's keep it the same). sys_reg defines only, the bits don't need the prefix.

I care less about the Python side, so no need to stick IMP_APL_ in front of everything in the regs file. Perhaps just amend the generator script so it adds that prefix on request or where required.

Reply:
I've added a flag to the generator script to add that IMP_APL prefix and fixed the defines